### DISCUSSION TITLE
Announcement: Temporarily Pausing New Issues and Pull Requests 🚧

### DISCUSSION THREAD

**Discussion by Abhilash107** [Abhilash107/Sidebar#65] [Announcements]
Hello, everyone! 👋

I am temporarily pausing the acceptance of new Issues and Pull Requests as I focus on the server side of the project. During this period, my priority will be improving the backend, performance, 

While I'm excited about the upcoming changes and enhancements, I kindly ask for your patience and understanding as I work behind the scenes.

Please stay tuned for updates—once this phase is complete, I'll reopen the repository for contributions and feedback.

Thank you for your support! 🙏

Best regards,
@Abhilash107

**Comment by Artemiskgg1** [comment]
okay! 👍
